.github/workflows/packaging.yml

```diff
@@ -1,17 +1,91 @@
 name: packaging
 on: [push, pull_request]
 
+env:
+  PACKAGE_SLEUTHKIT_JAVA_VERSION: 4.10.1
+  PACKAGE_SLEUTHKIT_JAVA_REL: 0
+  PACKAGE_SLEUTHKIT_JAVA_SHA: 44abd10602df70625bc438cf011e23ddc43a132a
+  PACKAGE_AUTOPSY_VERSION: 4.17.0
+  PACKAGE_AUTOPSY_REL: 1
+  PACKAGE_AUTOPSY_SHA: 39a2a13fa3cb7376acdd3125e862ace9626bde1b
+
 jobs:
+  arch-packages:
+    runs-on: ubuntu-latest
+    container:
+      image: archlinux
+      options: --privileged # Needed for building in clean chroot
+      volumes:
+        - /sys/fs/cgroup:/sys/fs/cgroup # systemd nspawn needs cgroups
+    steps:
+      - name: Install required dependencies
+        run: |
+          pacman -Syu --noconfirm
+          pacman -S git --noconfirm --needed --noprogressbar
+      - name: Clone sleuthkit-java build repo from AUR
+        run: |
+          git clone https://aur.archlinux.org/sleuthkit-java.git $HOME/sleuthkit-java
+          (cd $HOME/sleuthkit-java; git checkout $PACKAGE_SLEUTHKIT_JAVA_SHA)
+      - name: Build sleuthkit-java
+        uses: FFY00/build-arch-package@v1
+        with:
+          PKGBUILD: $HOME/sleuthkit-java/PKGBUILD
+          OUTDIR: /tmp/artifacts
+      - uses: actions/upload-artifact@v2
+        with:
+          name: arch-packages
+          path: /tmp/artifacts/*.pkg.tar.zst
+      - name: Clone autopsy-bin build repo from AUR
+        run: |
+          git clone https://aur.archlinux.org/autopsy-bin.git $HOME/autopsy-bin
+          (cd $HOME/autopsy-bin; git checkout $PACKAGE_AUTOPSY_SHA)
+      - name: Build autopsy-bin
+        run: |
+          chown user -R $HOME/autopsy-bin
+          cd $HOME/autopsy-bin
+          extra-x86_64-build -- -U user -I "$(find "/tmp/artifacts" -name 'sleuthkit-java-*.pkg.tar.zst')"
+          cp *.pkg.* /tmp/artifacts
```

Comment:
Why not `mv`?

Reply:
Initially, I had `mv` but changed `cp` for further debug. But it's really pointless, although I'll change it. Makes more sense

Reply:
Done